packages/visx-demo/src/components/icons/GitHub.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const GitHub = () => {
+  return (
+    <svg
+      width="100%"
+      height="100%"
+      viewBox="0 0 136 133"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      style={{
+        fill: '#515251',
+        fillRule: 'evenodd',
+        clipRule: 'evenodd',
+        strokeLinejoin: 'round',
+        strokeMiterlimit: 2,
+      }}
+    >
+      <g transform="matrix(1,0,0,1,-568.001,-447.669)">
+        <g transform="matrix(4.16667,0,0,4.16667,0,0)">
+          <g id="mini" transform="matrix(1,0,0,1,152.608,139.345)">
```

Comment:
should we remove the `id`s in the logo components since they create global variables?